kategory-effects/src/main/kotlin/kategory/effects/data/internal/IOFrame.kt

```diff
@@ -0,0 +1,35 @@
+package kategory.effects
+
+import kategory.Either
+
+interface IOFrame<in A, out R> : (A) -> R {
```

Comment:
Any way to make these private or protected so they are not abused in the IO public API?